### ISSUE TITLE
Accessory cardiac bronchus

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6632]
**Preferred term label:** Accessory cardiac bronchus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Accessory bronchus originating from the medial wall of the right or left ban bronchus or bronchus intermedius. A cardiac bronchus is usually blind-ended. PMID: 11158647

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology HP:0025426

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
